src/api_server/src/request/machine_configuration.rs

```diff
@@ -157,6 +166,35 @@ mod tests {
         {
             assert!(parse_put_machine_config(&Body::new(body)).is_err());
         }
+
+        // 5. Test that setting `ht_enabled: true` is successful on x86_64 while on aarch64, it is not.
```

Comment:
Can we also test that both true and false fail on aarch64?

Reply:
Actually only `ht_enabled=True` fails on ARM. `ht_enabled` defaults to False on all platforms.